scipy/stats/biasedurn.pyx

```diff
@@ -10,13 +10,13 @@ cdef class PyFishersNCHypergeometric:
 
     def mode(self):
         return self.c_fnch.mode()
-        
```

Comment:
These are because I was working in a text editor. My IDE, Spyder, seems to hold the files open so I can't build without closing it.